generate-wrapper.py

```diff
@@ -23,25 +23,67 @@ def typename(decl):
        raise ValueError(decl)
 
 class DuckDBHeaderVisitor(pycparser.c_ast.NodeVisitor):
-    result = ''
+    cpp_result = ''
+    types_result = ''
+    c_type_to_ts_type = {
+        "bool": "boolean",
+        "double": "number",
+        "char*": "string",
+        "char**": "out_string_wrapper",
+        "const char*": "string",
+        "const char**": "out_string_wrapper",
+        "float": "number",
+        "idx_t": "number",
+        "idx_t*": "idx_pointer",
+        "int8_t": "number",
+        "int16_t": "number",
+        "int32_t": "number",
+        "int64_t": "number", # should use bigint because max safe int in JS is 2^53-1
+        "uint8_t": "number",
+        "uint16_t": "number",
+        "uint32_t": "number",
+        "uint64_t": "number", # should use bigint because max safe int in JS is 2^53-1
+        "uint64_t*": "uint64_pointer",
+        "size_t": "number",
+        "void": "void",
+        "void*": "pointer",
+        "const void*": "pointer",
+    }
 
     def visit_TypeDecl(self, node):
         name = node.declname
         if not name.startswith('duckdb_'):
             return
 
         if isinstance(node.type, pycparser.c_ast.Struct):
-            self.result += f'exports.Set(Napi::String::New(env, "{name}"), duckdb_node::PointerHolder<{name}>::Init(env, "{name}")->Value());\n'
-
-        if isinstance(node.type, pycparser.c_ast.Enum):
-            self.result += f'auto {name}_enum = Napi::Object::New(env);\n'
+            self.cpp_result += f'exports.Set(Napi::String::New(env, "{name}"), duckdb_node::PointerHolder<{name}>::Init(env, "{name}")->Value());\n'
```

Comment:
I renamed "result" to "cpp_result", since there is now also a "types_result". I haven't changed the generated C++ at all.